skore/src/skore/_sklearn/_plot/metrics/confusion_matrix.py

```diff
@@ -71,6 +97,11 @@ def plot(
             conditions or all the population. If None, the confusion matrix will not be
             normalized.
 
+        threshold_value : float or None, default=None
+            The decision threshold to use when applicable.
+            If None and thresholds are available, plots the confusion matrix at the
+            default threshold (0.5).
```

Comment:
Actually it is 0.5 if the estimator expose a `predict_proba` but it is 0 if `decision_function` is used.

Indeed, it could be worth to use `SVC` classifier that only expose `decision_function` in the test to ensure the behaviour.